pages/_app/index.jsx

```diff
@@ -80,9 +77,11 @@ class App extends NextApp {
 
   fetchUser = async () => {
```

Comment:
@Joozty why is this function needed?

Reply:
It's among the first functions which are called when dashboard loads. It checks if a user is logged in or not. (Checking skipped when user navigates to pages which don't require log in.